From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-hugetlb-introduce-hugetlb_walk-checkpatch-fixes
Date: Fri Dec 16 09:19:24 AM PST 2022

ERROR: code indent should use tabs where possible
#267: FILE: mm/page_vma_mapped.c:172:
+                 * All callers that get here will already hold the$

ERROR: code indent should use tabs where possible
#268: FILE: mm/page_vma_mapped.c:173:
+                 * i_mmap_rwsem.  Therefore, no additional locks need to be$

ERROR: code indent should use tabs where possible
#269: FILE: mm/page_vma_mapped.c:174:
+                 * taken before calling hugetlb_walk().$

total: 3 errors, 0 warnings, 211 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/mm-hugetlb-introduce-hugetlb_walk.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Peter Xu <peterx@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/page_vma_mapped.c~mm-hugetlb-introduce-hugetlb_walk-checkpatch-fixes
+++ a/mm/page_vma_mapped.c
@@ -169,9 +169,9 @@ bool page_vma_mapped_walk(struct page_vm
 		if (pvmw->pte)
 			return not_found(pvmw);
 		/*
-                 * All callers that get here will already hold the
-                 * i_mmap_rwsem.  Therefore, no additional locks need to be
-                 * taken before calling hugetlb_walk().
+		 * All callers that get here will already hold the
+		 * i_mmap_rwsem.  Therefore, no additional locks need to be
+		 * taken before calling hugetlb_walk().
 		 */
 		pvmw->pte = hugetlb_walk(vma, pvmw->address, size);
 		if (!pvmw->pte)
_
